### PR TITLE
Fix typos in docs.

### PR DESCRIPTION
**Pull request by dongjoon-hyun**

### CODE REVIEW

File: docs/reference/search/profile.asciidoc [hunk 1]
```diff
@@ -261,7 +261,7 @@ The meaning of the stats are as follows:
 
     This parameter shows how long it takes to build a Scorer for the query.  A Scorer is the mechanism that
     iterates over matching documents generates a score per-document (e.g. how well does "foo" match the document?).
-    Note, this records the time required to generate the Scorer object, not actuall score the documents.  Some
+    Note, this records the time required to generate the Scorer object, not actual score the documents.  Some
```
Comment (nik9000) [inline docs/reference/search/profile.asciidoc]: I think this should be *actually*.

Comment (dongjoon-hyun) [inline docs/reference/search/profile.asciidoc]: Oh, thank you, @nik9000 . You're right. I'll fix it.

**Comment by dongjoon-hyun** [comment]
Thank you!

**Comment by nik9000** [comment]
Merged! Thanks @dongjoon-hyun!

Backported to 2.x with ae02ed549f5459e86de38f9ae1cd41edec79adb3 and da63bddf0011210854d346e897f8ebd878a7ece3.

Backproted to 2.2 bc051c1ba62346ee507211819fb87152cf68c536 and bb3a0c2463ddb9ccc48f5fed1fb2f70fddfe3bc4.

**Pull request merged**